demonstrations/tutorial_vqe_spin_sectors.py

```diff
@@ -0,0 +1,349 @@
+r"""
+VQE in different spin sectors
+=============================
+
+.. meta::
+    :property="og:description": Find the lowest-energy states of a Hamiltonian in different spin sectors
+    :property="og:image": https://pennylane.ai/qml/_images/thumbnail_spectra_h2.png
+
+.. related::
+   tutorial_vqe Variational Quantum Eigensolver
+   tutorial_vqe_parallel VQE with parallel QPUs
+
+*Author: PennyLane dev team. Last updated: 25 June 2021.*
+
+The Variational Quantum Eigensolver (VQE) algorithm has proven to be a valuable approach
+to find the lowest-energy eigenstate of the electronic Hamiltonian of a molecule using a
```

Comment:
```suggestion
for finding the lowest-energy eigenstate of the electronic Hamiltonian of a molecule using a
```
Consider also splitting this sentence into two, as it's a bit long.